roles/zuul/meta/main.yml

```diff
@@ -1,2 +1,12 @@
 dependencies:
-  - pip
+  - role: python-app
+    name: zuul
+    python_pipdeps:
+      - name: jenkins-job-builder
+        version: '1.6.1'
+      - name: pyzmq
+      - name: ansible
+      - name: statsd
+    git_repo: "{{ zuul_git_repo_url }}"
+    git_version: "{{ zuul_git_branch }}"
+    python_notify: Restart zuul
```

Comment:
https://travis-ci.org/BonnyCI/hoist/jobs/199414443#L1544 Looks like the handler is getting called